frontend/src/metabase/containers/SavedQuestionLoader.jsx

```diff
@@ -38,107 +39,50 @@ import Question from "metabase-lib/Question";
  * The raw un-connected component is also exported so we can unit test it
  * without the redux store.
  */
```

Comment:
Nit: I think we can remove this comment since now we are using Redux inside the component